back/lib/tasks/single_use/20251029_delete_legacy_file_records.rake

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+namespace :single_use do
+  desc 'Delete all legacy file records after migration to new Files system'
+  # This rake task deletes records explicitly using +destroy!+ before we delete the tables
+  # in a migration in order to run the destroy callbacks and clean up associated files on
+  # S3.
```

Comment:
Don't recall the history behind this. Have all legacy files been copied to the new model?

Reply:
Yes :)